assets/js/components/notifications/CoreSiteBannerNotifications.js

```diff
@@ -75,30 +70,14 @@ const CoreSiteBannerNotifications = () => {
 		return null;
 	}
 
-	return notifications.map( ( notification ) => (
-		<BannerNotification
-			key={ notification.id }
-			id={ notification.id }
-			title={ notification.title || '' }
-			description={ notification.content || '' }
-			learnMoreURL={ notification.learnMoreURL || '' }
-			learnMoreLabel={ notification.learnMoreLabel || '' }
-			ctaLink={ notification.ctaURL || '' }
-			ctaLabel={ notification.ctaLabel || '' }
-			ctaTarget={ notification.ctaTarget || '' }
-			dismiss={
-				notification.dismissLabel ||
-				__( 'OK, Got it!', 'google-site-kit' )
-			}
-			isDismissible={ notification.dismissible }
-			onCTAClick={ async () => {
-				await acceptNotification( notification.id );
-			} }
-			onDismiss={ async () => {
-				await dismissNotification( notification.id );
-			} }
-		/>
-	) );
+	return notifications.map( ( notification ) => {
+		return (
+			<CoreSiteBannerNotification
+				key={ notification.id }
+				notification={ notification }
```

Comment:
Let's pass only required properties to the child component:

```js
return notifications.map( ( { id, title, content, ... }  ) => {
    return (
        <CoreSiteBannerNotification
            key={ id }
            id={ id }
            title={ title }
            content={ content }
            learnMoreURL={ ... }
            ...
        />
} );
```

Reply:
https://github.com/google/site-kit-wp/pull/4349/commits/935171334551e7fb64936825294273350989236a